specification/cognitiveservices/OpenAI.Inference/models/images/images.tsp

```diff
@@ -10,80 +10,170 @@ using TypeSpec.Versioning;
 
   @doc("Image generation response items should provide image data as a base64-encoded string.")
-  Base64: "b64_json",
+  base64: "b64_json",
 }
 
 @doc("Represents the request data used to generate images.")
 @added(ServiceApiVersions.v2023_06_01_Preview)
 model ImageGenerationOptions {
+  @doc("The model to use for image generation.")
+  `model`?: string;
+
   @doc("A description of the desired images.")
   prompt: string;
 
-  @doc("The number of images to generate (defaults to 1).")
+  @doc("""
+  The number of images to generate.
+  Dall-e-2 models support values between 1 and 10.
+  Dall-e-3 models only support a value of 1.
+  """)
   @projectedName("csharp", "ImageCount")
   n?: int32 = 1;
 
-  @doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024 (defaults to 1024x1024).")
-  size?: ImageSize = ImageSize.Size1024x1024;
-
   @doc("""
-    The format in which image generation response items should be presented.
-    Azure OpenAI only supports URL response items.
+  The desired dimensions for generated images.
+  Dall-e-2 models support 256x256, 512x512, or 1024x1024.
+  Dall-e-3 models support 1024x1024, 1792x1024, or 1024x1792.
   """)
+  size?: ImageSize = ImageSize.size1024x1024;
+
+  @doc("The format in which image generation response items should be presented.")
   @projectedName("json", "response_format")
-  responseFormat?: ImageGenerationResponseFormat;
+  responseFormat?: ImageGenerationResponseFormat = ImageGenerationResponseFormat.url;
+
+  @doc("""
+  The desired image generation quality level to use.
+  Only configurable with dall-e-3 models.
+  """)
+  @added(ServiceApiVersions.v2023_12_01_Preview)
+  quality?: ImageGenerationQuality = ImageGenerationQuality.standard;
+
+  @doc("""
+  The desired image generation style to use.
+  Only configurable with dall-e-3 models.
+  """)
+  @added(ServiceApiVersions.v2023_12_01_Preview)
+  style?: ImageGenerationStyle = ImageGenerationStyle.vivid;
 
   @doc("A unique identifier representing your end-user, which can help to monitor and detect abuse.")
   user?: string;
 }
 
-@doc("An image response item that provides a URL from which an image may be accessed.")
+@doc("""
+A representation of a single generated image, provided as either base64-encoded data or as a URL from which the image
+may be retrieved.
+""")
 @added(ServiceApiVersions.v2023_06_01_Preview)
-model ImageLocation {
+model ImageGenerationData {
   @doc("The URL that provides temporary access to download the generated image.")
-  url: url;
-}
+  url?: url;
 
-@doc("An image response item that directly represents the image data as a base64-encoded string.")
-@added(ServiceApiVersions.v2023_06_01_Preview)
-model ImagePayload {
-  @doc("The complete data for an image represented as a base64-encoded string.")
+  @doc("The complete data for an image, represented as a base64-encoded string.")
   @projectedName("json", "b64_json")
-  base64Data: string;
+  base64Data?: string;
```

Comment:
Not entirely sure if worth the breaking change, but just in case, we could match the field name here to what we have for `ImageGenerationResponseFormat` and just go with `base64`